doc/man/8/cephadm.rst

```diff
@@ -194,6 +194,16 @@ Arguments:
 * [--skip-firewalld]           Do not configure firewalld
 * [--skip-pull]                do not pull the latest image before adopting
 
+Configuration:
+
+When starting the shell, cephadm looks for configuration in the following order:
+
+1. An explicit, user provided path to a config file (``-c/--config`` option)
+2. Config file for daemon specified with ``--name`` parameter (``/var/lib/ceph<fsid>/<daemon-name>/config``)
+3. ``/var/lib/ceph/<fsid>/config/ceph.conf`` if it exists
+4. The config file for a ``mon`` daemon (``/var/lib/ceph/<fsid>/mon.<mon-id>/config``) if it exists
```

Comment:
This may be pedantic, but IMHO looking for config in a listed order to me sort of implies "if it exists", so perhaps that string in the above two lines is superfluous.

Since software varies, I'd like to see an explicit mention of whether this order is first-fit, last-fit, cumulative, or something else.

Reply:
> This may be pedantic, but IMHO looking for config in a listed order to me sort of implies "if it exists", so perhaps that string in the above two lines is superfluous.

Agree, I'll remove it.

> 
> Since software varies, I'd like to see an explicit mention of whether this order is first-fit, last-fit, cumulative, or something else.

Well, the order here is _priority_ order. I mean if all of these options exist the order would be the specified in these entries. 

Is it OK to say something like (adding `priority` word)?

`When starting the shell, cephadm looks for configuration in the following priority order:
`

Reply:
What I mean is does it pick only the first match it finds?

Reply:
Yes, it will use the first match if finds following the specified order.